src/clj/witan/send/model/prepare.clj

```diff
@@ -230,9 +274,7 @@
                               (transitions-map modified-transitions)
                               (transitions-map transitions))
          transitions-filtered (when filter-transitions-from
-                                (mapcat (fn [year] (filter #(> (:calendar-year %) year)
-                                                           (or modified-transitions transitions)))
-                                        [filter-transitions-from]))
+                                (reduce #(sequence (build-filter %2) %1) (or modified-transitions transitions) filter-transitions-from))
```

Comment:
that line is a bit of a mind bender. It is using a seq as a reducing function over `filtered-transitions-from` as input and using `(or modified-transitions transition)` as the seed state?

Reply:
not quite, it's using `filtered-transitions-from` as input to build a reducing function to then apply to `(or modified-transitions transition)`.